deployment/clusterObjectModel/mainParser/machine.py

```diff
@@ -141,8 +141,10 @@ def run(self):
                     host["keyfile-path"] = com_machine["default-machine-properties"]["keyfile-path"]
             if "nodename" not in host:
                 host["nodename"] = host["hostip"]
-            if "docker-data" not in host:
+            if "docker-data" not in host and "docker-data-root" not in com_machine["machine-sku"][host["machine-type"]]:
```

Comment:
What's the difference of `docker-data` and `docker-data-root`?

Reply:
the configuration in docker-d is named data-root. I have no idea when the docker-data is added into machine configuraiton.